sources/packages/backend/apps/api/src/route-controllers/student/_tests_/e2e/student.institutions.controller.getStudentApplicationSummary.e2e-spec.ts

```diff
@@ -0,0 +1,191 @@
+    await authorizeUserTokenForLocation(
+      appDataSource,
+      InstitutionTokenTypes.CollegeCUser,
+      collegeCLocation,
+    );
+    applicationRepo = appDataSource.getRepository(Application);
+    studentAssessmentRepo = dataSource.getRepository(StudentAssessment);
+    educationProgramOfferingRepo = dataSource.getRepository(
+      EducationProgramOffering,
+    );
+    institutionLocationRepo = dataSource.getRepository(InstitutionLocation);
+  });
+
+  it("Should get the student application details as summary when student has a submitted application for the institution (application with location id saved).", async () => {
+    // Arrange
+
+    // Student has a submitted application to the institution.
+    const student = await saveFakeStudent(appDataSource);
+    const application = createFakeApplication({
+      location: collegeCLocation,
+      student,
+    });
+    const savedApplication = await applicationRepo.save(application);
+    // Original assessment.
+    const fakeOriginalAssessment = createFakeStudentAssessment({
+      auditUser: student.user,
+    });
+    fakeOriginalAssessment.application = savedApplication;
+    // Offering.
+    const fakeOffering = createFakeEducationProgramOffering({
+      auditUser: student.user,
+    });
+    fakeOffering.offeringIntensity = OfferingIntensity.fullTime;
+    const savedOffering = await educationProgramOfferingRepo.save(fakeOffering);
+    fakeOriginalAssessment.offering = savedOffering;
+    const savedOriginalAssessment = await studentAssessmentRepo.save(
+      fakeOriginalAssessment,
+    );
+    savedApplication.currentAssessment = savedOriginalAssessment;
+    await applicationRepo.save(savedApplication);
+
+    const endpoint = `/institutions/student/${student.id}/application-summary?page=0&pageLimit=10`;
+    const institutionUserToken = await getInstitutionToken(
+      InstitutionTokenTypes.CollegeCUser,
+    );
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .get(endpoint)
+      .auth(institutionUserToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK)
+      .expect({
+        results: [
+          {
+            id: application.id,
+            applicationNumber: application.applicationNumber,
+            studyStartPeriod:
+              application.currentAssessment.offering.studyStartDate,
+            studyEndPeriod: application.currentAssessment.offering.studyEndDate,
+            applicationName: "Financial Aid Application",
+            status: application.applicationStatus,
+          },
+        ],
+        count: 1,
+      });
+  });
+
+  it("Should get an empty array as application summary when student has a draft application for the institution (application without the location id saved(i.e draft/cancelled draft status)).", async () => {
+    // Arrange
+
+    // Student has a draft application to the institution.
+    const student = await saveFakeStudent(appDataSource);
+    const application = createFakeApplication({
```

Comment:
. I understand the intention of going deeper but the authorization will definitely validate for a linked application where on a happy path the result could never be empty. IMO this scenario is not required.

Reply:
I got your point, we can do either one 
1. add one more application to the institution that belongs to the student and expect only the application that belongs to the institution? 
2. or keep it as it is and adjust the except (to return a forbidden error, I believe),  once you merge public institution authorization logic. 

I prefer the 2nd option thoughts? @andrewsignori-aot @andrepestana-aot @guru-aot

Reply:
If option 1 is not covered yet by any other E2E I would prefer it @ann-aot because the decorator will not ensure that and it would be good to have it asserted here.

Reply:
ya. @andrewsignori-aot  first is already covered in the first TC. So, i will remove these

Reply:
@ann-aot I am not able to see the scenario where student has applications for more than one institution and the list returns only application which belongs to the particular institution.